### ISSUE TITLE
Not able to navigate to methods after recent language-php versions

### ISSUE THREAD

**Issue by daltones** [php-integrator/atom-navigation#42]
After some recent updates, navigating to methods (both instance or static) won't work.

Like in this example:
```php
<?php

class Foo {
  public function bar() {}
  
  public static function baz() {}
}

$foo = new Foo();
$foo->bar();

Foo::baz();
```

Navigation to methods `bar()` and `baz()` is not available from `->bar()` and `::baz()`. (Notice that navigating to class `Foo` still works).

If I have understood correctly, [this commit in `language-php`](https://github.com/atom/language-php/commit/0af78afdfa9a6c48333ad1ddbfb58f7e140f8545) have broken things here.

`MethodProvider` for hyperclick expects a scope containing `function-call` ([here](https://github.com/php-integrator/atom-navigation/blob/master/lib/MethodProvider.coffee)). But now `language-php` emits scopes such as `meta.method-call.php` or `meta.method-call.static.php`.

**Comment by tortuetorche** [comment]
Hi @daltones ,

Thank you for these explanations!
The only workaround I found, on Windows, is to downgrade Atom to version [1.19.7](https://github.com/atom/atom/releases/download/v1.19.7/AtomSetup-x64.exe)

Have a good day,
Tortue Torche

**Comment by Gert-dev** [comment]
Yes, I'm aware of this and suspected this day would come. It's my own fault for depending on Atom's CSS classes. Since I was planning to move more functionality to the core anyway, I've reimplemented it there and it will be part of 3.1 [1].

[1] https://gitlab.com/php-integrator/core/issues/42#note_41035418

**Comment by daltones** [comment]
Thanks @Gert-dev! I'm excited for the next release.

And thanks again for this amazing tool! It really helps PHP people :)

**Comment by Gert-dev** [comment]
I've just released base package 3.1.0 with core 3.1.0 that includes the new functionality!
